dali/pipeline/operators/crop/bbox_crop.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/crop/bbox_crop.h"
+
+namespace dali {
+
+DALI_SCHEMA(BBoxCrop)
```

Comment:
What BBox format do we support? #110 and #188 support ltrb and xywh.

Reply:
accepted. Supporting xywh at the moment.